channels/applicationinsights-channel-js/src/Interfaces.ts

```diff
@@ -94,6 +95,24 @@ export interface ISenderConfig {
      * @since 3.0.1
      */
     enableSendPromise?: boolean;
+
+    /**
+     * [Optional] The HTTP override that should be used to send requests, as an IXHROverride object.
+     * By default during the unload of a page or if the event specifies that it wants to use sendBeacon() or sync fetch (with keep-alive),
+     * this override will NOT be called.
+     * If onunloadDisableFetch and onunloadDisableBeacon are true, You can now change this behavior by enabling the 'alwaysUseXhrOverride' configuration value.
+     * The payload data (first argument) now also includes any configured 'timeout' (defaults to undefined) and whether you should avoid
+     * creating any synchronous XHR requests 'disableXhr' (defaults to false/undefined)
+     */
+    httpXHROverride?: IXHROverride;
```

Comment:
Please also add the `@since` so that the generated typedoc will have the correct version (when we fully publish the docs)